brownie/_expansion.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from dotenv.variables import parse_variables
+
+
+def expand_posix_vars(dct, variables):
```

Comment:
Took me a moment to understand what `dct` and `variables` are here.  Some more expressive naming, mypy types, and/or an "arguments" section in the natspec might help.